README.md

```diff
@@ -473,6 +473,20 @@ All statements above will return false if used with `===`
 
 <p align="right"><a href="#table-of-contents">⬆ Return to top</a></p>
 
+## ![✔] 4.9 Refactor regularly using static analysis tools
+
+**TL;DR:** Using static analysis tools helps by giving objective ways to improve code quality and keep your code maintainable. Keeping out code smells and bloat will make maintenance easier and provides better readability to your code.
+
+Here are some tools to automate finding code to improve:
+  - [Sonarqube](https://www.sonarqube.org/) (2,600+ [stars](https://github.com/SonarSource/sonarqube))
```

Comment:
@TheHollidayInn For the sake of consistent formatting, I would move this section inside or make it a plain line with two links: see bullet 3.1